src/utils/chunks/upload.rs

```diff
@@ -24,4 +34,320 @@ pub trait ChunkOptions {
 
     /// Returns the server options for the chunk upload.
     fn server_options(&self) -> &ChunkServerOptions;
+
+    /// Uploads the given chunked objects to the server.
+    fn upload_chunked_objects<T>(self, chunked: &[Chunked<T>]) -> Result<(Vec<DebugInfoFile>, bool)>
```

Comment:
I wouldn't do this as a method on this trait; it seems to me that this should be a free function, although this is admittedly based on an aesthetic feeling.